### DISCUSSION TITLE
Need help with Github Actions

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#1218] [General]
I checked in some changes to the Celsius function, and (hand-waving begins) it appears that requires a change to some Rust code that generates new test cases (???)

The failing test is at:

https://github.com/richb-hanover/prql/actions/runs/3609073844/jobs/6082048039#step:12:974

What should I do to clean up the commit? Many thanks.

**Comment by max-sixty** [comment]
Great — `cargo insta test --accept` will replace all the snapshots!

For reference, running `task test-all` will cover everything, including that command.
